src/background/telemetry/metrics.js

```diff
@@ -192,7 +194,18 @@ export default class Metrics {
       // Date of install (former install_date)
       buildQueryPair('id', this.storage.installDate) +
       // Toolbar pinned
-      buildQueryPair('tp', Number(conf.userSettings?.isOnToolbar ?? -1));
+      buildQueryPair('tp', Number(conf.userSettings?.isOnToolbar ?? -1)) +
+      // ZAP mode (-1 = flag disabled, 0 = default, 1 = zap, 2 = switched from zap to default)
```

Comment:
"2" also includes mode set to zap, but `mode` was touched. It is worth mentioning in the comment.